src/middlewared/middlewared/plugins/chart_releases_linux/scale_workload.py

```diff
@@ -278,3 +278,13 @@ async def scale_down_resources_consuming_locked_paths(self):
         ]
         if args:
             await self.middleware.call('core.bulk', 'chart.release.scale', args)
+
+    @private
+    async def wait_for_namespace_to_terminate(self, namespace, timeout=300):
+        # wait for release to uninstall properly, helm right now does not support a flag for this but
+        # a feature request is open in the community https://github.com/helm/helm/issues/2378
```

Comment:
My bad..
It does not apply to namespace as we create namespace and not helm. I had copied the function signature from what we used to do for pods.
I am working on making a different PR where we consume that flag though but would still likely keep our various logic in place until we are sure upstream has no issues because it appears there are some edge cases which are not covered.
Removed the comment as it is not valid ✅